tests/e2e/test_rosacli_cluster.go

```diff
@@ -332,6 +332,145 @@ var _ = Describe("Edit cluster",
 					To(ContainSubstring(
 						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))
 			})
+
+		// Excluded until bug on OCP-73161 is resolved
+		It("can verify delete protection on a rosa cluster - [id:73161]",
+			labels.High, labels.Runtime.Day2, labels.Exclude,
+			func() {
+				By("Enable delete protection on the cluster")
+				deleteProtection := constants.DeleteProtectionEnabled
+				_, err := clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=true",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Check the enable result from cluster description")
+				output, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				clusterDetail, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(clusterDetail.EnableDeleteProtection).To(Equal(deleteProtection))
+
+				By("Attempt to delete cluster with delete protection enabled")
+				defer func() {
```

Comment:
Why using defer func here, it will be executed after the delete protection is disabled on the cluster, just try to delete it after you enable the delete protection